---
layout: page
title: CV
subtitle: 
use-site-title: false
---
------------------------------
**Education** 
<br>
**2016-Present**
<br>
PhD. Candidate, Arking lab   
<br>
My work in the Arking lab focuses on linear mixed models to understand the relationships between mitochondrial DNA copy number and nuclear gene expression.  I also work on elucidating the relationships between mitochondrial DNA heteroplasmy and aging-related diseases.  
<br>
Johns Hopkins University School of Medicine, Baltimore, MD  
McKusick-Nathans Institute of Genetic Medicine  

**2016-2017**
<br>
*Rotation student, Cutting lab*
<br>
Compared normalization methods for RNA-sequencing using publicly available data from recount2.  
<br>
*Rotation student, Goff lab*
<br>
Analyzed a single-cell RNA sequencing dataset using Monocle with the goal of understanding cortical development in mice and built shiny apps to allow for easy data visualization for a collaborator.  
<br>
*Rotation student, Lloyd lab*
<br>
Identified genes of interest and performed a fly screen to examine potential modifier genes for C9 ALS.  
<br>
*Rotation student, Arking lab*
<br>
Used plink to perform quality control on 1000 genomes data, extracted SNPs that separated certain ethnic populations, and evaluated dimensionality reduction methods.  
<br>
**2015-2016**  
*NIH Postbac IRTA* 
<br>
Utilized the CRISPR/Cas9 system to edit induced pluripotent stem cells.  
<br>
<br>

**2011-2015**  
A.B. in Chemistry, Magna cum laude, ACS certified  
Bryn Mawr College, Bryn Mawr, PA   

**2013-2015**  
*Undergraduate Research Assistant, Bryn Mawr College*
<br>
Synthesized ruthenium polypyridyl complexes and evaluated DNA intercalation and photocleavage ability.  
<br>
<br>
**2013**  
*Intern, Naval Medical Research Center*
<br>
Tested and compared sensitivity and specificity of primer sets for Trypanosoma cruzi.  
		
